Update cloudgateway_change_log.md (#4052)
SkylineCommunications/dataminer-docs: release-notes/DxMs/cloudgateway_change_log.md

# Cloud Gateway change log

#### 10 December 2024 - Enhancement - CloudGateway 2.16.0 - Dependencies updated [ID 41681]

> An important dependency has been updated where the cloud connection is based on to proxy most remote access requests over.

Reviewer: ```suggestion
An important dependency has been updated that the dataminer.services connection uses to proxy most remote access requests.
```
